idact/detail/allocate_resources_app/main.py

```diff
@@ -0,0 +1,144 @@
+"""This module contains the :func:`main` function for the allocating resources
```

Comment:
Please change doc a little bit, something like _for resources allocation for the future_